js/src/util/index.js

```diff
@@ -135,20 +135,33 @@ const typeCheckConfig = (componentName, config, configTypes) => {
 }
 
 const isVisible = element => {
-  if (!element) {
+  const eligibleElement = el => el && el.style
+
+  if (!eligibleElement(element)) {
+    return false
+  }
+
+  if (element.tagName !== 'AREA' && element.getClientRects().length === 0) {
```

Comment:
I think SVG too, returns empty `getClientRects()`

Reply:
Well at least not in Chrome.. even a `<clipPath>` is returning one.

Reply:
after a second thought, mayby we overthink it. It is for internal use, and by default checks buttons and component elements. So maybe we can stick on basics. Nor svg/area/clippath etc .

it we find an edge case we can support it later. Either way, the check is better than before :smile: